telco-examples/mgmt-cluster/eib/network/mgmt-cluster-network.yaml

```diff
@@ -0,0 +1,25 @@
+routes:
+  config:
+  - destination: 0.0.0.0/0
+    metric: 100
+    next-hop-address: ${MGMT_GATEWAY}
+    next-hop-interface: eth0
+    table-id: 254
+dns-resolver:
+  config:
+    server:
+    - ${MGMT_DNS}
+    - 8.8.8.8
+interfaces:
+- name: eth0
+  type: ethernet
+  state: up
+  mac-address: aa:aa:aa:bb:bb:22
```

Comment:
Should we use a variable for the MAC address as well?

Reply:
yes, you're right. Thanks